packages/stream_chat/lib/src/core/models/poll.dart

```diff
@@ -109,20 +109,20 @@ class Poll extends Equatable {
   final Map<String, int> voteCountsByOption;
 
   /// Map of latest votes by option.
-  @JsonKey(name: 'latest_votes_by_option', includeToJson: false)
-  final Map<String, List<PollVote>> votesByOption;
+  @JsonKey(includeToJson: false)
+  final Map<String, List<PollVote>> latestVotesByOption;
 
   /// List of votes received by the poll.
   ///
   /// Note: This does not include the answers provided by the users,
-  /// see [answers] for that.
-  List<PollVote> get votes => [
-        ...votesByOption.values.flattened.where((it) => !it.isAnswer),
+  /// see [latestAnswers] for that.
+  List<PollVote> get latestVotes => [
+        ...latestVotesByOption.values.flattened.where((it) => !it.isAnswer),
```

Comment:
Does it make sense to declare this field as `final`, given that it is based on another `final` field? :)

Reply:
Yes, makes sense. Done